### ISSUE TITLE
OPTIONS fake event eat errors

### ISSUE THREAD

**Issue by mrbuds** [WeakAuras/WeakAuras2#1482]
Code run with event "OPTIONS" doesn't trigger an error for bugsack

example, create a TSU with this code
```
function()
    print("TEST BEFORE")
    prit("OPTIONS!", i)
    math.sdff(123) 
    print("TEST AFTER")
    
    return true
end
```
it print "TEST BEFORE", but no error

**Comment by Stanzilla** [comment]
That is pretty annoying that it eats subsequent errors, we should not do that.

**Comment by InfusOnWoW** [comment]
That's intended. The OPTIONS event is a pretty breaking change, so I had to disable error messages from while in the options. I don't like that, but I don't have a better idea on how to introduce that change.

**Comment by emptyrivers** [comment]
I think that this would be nicely solved with #1208. Then, the error would not be eaten, but it would also not necessarily annoy end users who log in one day to find that everything is broken horribly and they don't understand what's going on. Instead they could see that a specific function is broken.
